html_reference.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<link rel="stylesheet" href="css_stylesheet.css">
+		<script src="https://kit.fontawesome.com/c90803ff49.js" crossorigin="anonymous"></script>
+		<title>Milo Girardet</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+	</head>
+	<body onTouchStart="">
```

Comment:
remove the onTouchStart attribute since it's not doing anything.

Reply:
Alex mentioned that's how you get the _:active_ to work on an iPhone.

Reply:
Fair enough, it does the job!